### ISSUE TITLE
compiled directory not created for jinja target alone

### ISSUE THREAD

**Issue by ademariag** [deepmind/kapitan#64]
Kapitan is failing to create directories for Jinja targets.

```
$ kapitan compile -v -t global_tools
2018-04-06 23:54:41,982 gnupg        DEBUG    231982: gpg --status-fd 2 --no-tty --debug ipc --fixed-list-mode --batch --with-colons --version
2018-04-06 23:54:41,983 gnupg        DEBUG    stderr reader: <Thread(Thread-1, initial daemon)>
2018-04-06 23:54:41,983 gnupg        DEBUG    stdout reader: <Thread(Thread-2, initial daemon)>
2018-04-06 23:54:41,985 gnupg        DEBUG    gpg: reading options from '/usr/local/google/home/ademaria/.gnupg/gpg.conf'
2018-04-06 23:54:41,986 gnupg        DEBUG    chunk: b'gpg (GnuPG) 2.2.2\nlibgcrypt 1.8.1\nCopyright (C) 2017 Free Software Foundation, Inc.\nLicense GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>\nThis is free software: you are free to change and redistribute it.\nThere is NO WARRANTY, to t'
2018-04-06 23:54:41,995 kapitan.resources DEBUG    Using reclass inventory config at: ./inventory/reclass-config.yml
2018-04-06 23:54:43,258 kapitan.targets DEBUG    load_target_inventory: found valid kapitan target global_tools
2018-04-06 23:54:43,263 kapitan.resources DEBUG    Using reclass inventory config at: /usr/local/google/home/ademaria/projects/health-templates/inventory/reclass-config.yml
2018-04-06 23:54:44,586 kapitan.utils DEBUG    render_jinja2 rendering /usr/local/google/home/ademaria/projects/health-templates/scripts/global_tools/create_secrets.sh
2018-04-06 23:54:44,594 kapitan.targets DEBUG    Wrote /tmp/tmpe0lyg75a.kapitan/global_tools/scripts/create_secrets.sh with mode 0750
2018-04-06 23:54:44,595 kapitan.targets INFO     Compiled global_tools (1.34s)
2018-04-06 23:54:44,645 kapitan.targets DEBUG    Compile pool terminated
2018-04-06 23:54:44,645 kapitan.targets ERROR
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/usr/local/google/home/ademaria/kapitan/lib/python3.6/site-packages/kapitan/targets.py", line 76, in compile_targets
    shutil.rmtree(compile_path_target)
  File "/usr/local/google/home/ademaria/kapitan/lib/python3.6/shutil.py", line 471, in rmtree
    onerror(os.lstat, path, sys.exc_info())
  File "/usr/local/google/home/ademaria/kapitan/lib/python3.6/shutil.py", line 469, in rmtree
    orig_st = os.lstat(path)
FileNotFoundError: [Errno 2] No such file or directory: './compiled/global_tools'
2018-04-06 23:54:44,648 kapitan.targets DEBUG    Removed /tmp/tmpe0lyg75a.kapitan
```
```
$ mkdir -p compiled/global_tools
$ kapitan compile -v -t global_tools
2018-04-06 23:55:26,086 gnupg        DEBUG    232115: gpg --status-fd 2 --no-tty --debug ipc --fixed-list-mode --batch --with-colons --version
2018-04-06 23:55:26,087 gnupg        DEBUG    stderr reader: <Thread(Thread-1, initial daemon)>
2018-04-06 23:55:26,087 gnupg        DEBUG    stdout reader: <Thread(Thread-2, initial daemon)>
2018-04-06 23:55:26,091 gnupg        DEBUG    gpg: reading options from '/usr/local/google/home/ademaria/.gnupg/gpg.conf'
2018-04-06 23:55:26,091 gnupg        DEBUG    chunk: b'gpg (GnuPG) 2.2.2\nlibgcrypt 1.8.1\nCopyright (C) 2017 Free Software Foundation, Inc.\nLicense GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>\nThis is free software: you are free to change and redistribute it.\nThere is NO WARRANTY, to t'
2018-04-06 23:55:26,106 kapitan.resources DEBUG    Using reclass inventory config at: ./inventory/reclass-config.yml
2018-04-06 23:55:27,380 kapitan.targets DEBUG    load_target_inventory: found valid kapitan target global_tools
2018-04-06 23:55:27,385 kapitan.resources DEBUG    Using reclass inventory config at: /usr/local/google/home/ademaria/projects/health-templates/inventory/reclass-config.yml
2018-04-06 23:55:28,754 kapitan.utils DEBUG    render_jinja2 rendering /usr/local/google/home/ademaria/projects/health-templates/scripts/global_tools/create_secrets.sh
2018-04-06 23:55:28,764 kapitan.targets DEBUG    Wrote /tmp/tmplnho7_0j.kapitan/global_tools/scripts/create_secrets.sh with mode 0750
2018-04-06 23:55:28,764 kapitan.targets INFO     Compiled global_tools (1.38s)
2018-04-06 23:55:28,765 kapitan.targets DEBUG    Copied /tmp/tmplnho7_0j.kapitan/global_tools into ./compiled/global_tools
2018-04-06 23:55:28,765 kapitan.targets DEBUG    Removed /tmp/tmplnho7_0j.kapitan
```

**Comment by ramaro** [comment]
I can't reproduce this using the latest release.

**Comment by adrianchifor** [comment]
Can't reproduce. I believe this was fixed by #51. Will close for now.
